### ISSUE TITLE
Request: auto exposure setting

### ISSUE THREAD

**Issue by doktorsleepelss** [emoose/DLSSTweaks#3]
This would be nice to control because some games have it off by default. I know people like using the dev dll to turn it on for RDR2 becuse hair looks better. I've also noticed it reduces ghosting in some games that have it off like A Plague Tale Requiem.

**Comment by emoose** [comment]
Might have found a way to enable it, dev overlay seems to show auto-exposure is turned on with it at least, could you give this build a try? (change the ForceAutoExposure setting in the INI)

[dlsstweaks-0.123.7.zip](https://github.com/emoose/DLSSTweaks/files/10726014/dlsstweaks-0.123.7.zip)

**Comment by doktorsleepelss** [comment]
Yup, defintely works. Thanks. There's an area in a Plauge Tale Requiem with dust particles that ghost with autoexposure off.  I could see the difference when I turn it on even with the non-dev dll.

**Comment by emoose** [comment]
Nice thanks for trying it, a reddit user also confirmed it working for them too, posted a new release at https://github.com/emoose/DLSSTweaks/releases/tag/0.123.7

BTW a couple people posted on reddit that it doesn't work on RDR2 for them, could you test whether the latest ver still works on your copy? Also which store did you get yours from btw? I think some store versions might have issues with loading DLLs like this..

**Comment by doktorsleepelss** [comment]
It was the steam version, but yeah, they said changing it to DX12 fixed it.
